content/en/docs/reference/glossary/workload.md

```diff
@@ -1,28 +1,29 @@
 ---
-title: Workload
-id: workload
-date: 2019-02-12
-full_link: /docs/concepts/workloads/
+title: Workloads
+id: workloads
+date: 2019-02-13
+full_link: https://kubernetes.io/docs/concepts/workloads/
 short_description: >
-  A set of applications for processing information to serve a purpose that is valuable to a single user or group of users.
+   A workload is a framing term to refer to a set of APIs/resources/objects.
```

Comment:
Sorry, I used "framing term" as a way of redirecting the conversation. It wasn't meant as the language for the glossary entry. See the API refdocs: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#
And you might want to explore the language of workloads elsewhere in the docs a bit further too.

Reply:
done